.gitmodules

```diff
@@ -1,6 +1,3 @@
 [submodule "contrib/protostellar"]
```

Comment:
Should this use goprotostellar now as well?

Reply:
Yes, but I'll do that in a different PR.